justfile

```diff
@@ -4,8 +4,8 @@ export CONFIG_DIR := env_var_or_default('CONFIG_DIR', '$HOME/.krustlet/config')
 
 run: run-wascc
 
-build:
-    cargo build
+build +FLAGS='':
```

Comment:
All of these can be replaced with the new `*FLAGS` variadic arg once a new release of `just` is cut (it is currently on master)